docs/ui.md

```diff
@@ -17,4 +17,28 @@ You can also use other web servers to serve the contents of the www/app director
 ### Running remotely
 When Kubernetes is deployed remotely, the UI is deployed as a cluster addon. To access it, visit `/ui`, which redirects to `/api/v1/proxy/namespaces/default/services/kube-ui/#/dashboard/`, on your master server.
 
+## Using the UI
+Kubernetes UI can be used to introspect your current cluster, such as checking how resources are used, or looking at error messages. You cannot, however, use the UI to modify your cluster. 
+
+### Node Resource Usage 
+After accessing Kubernetes UI, you'll see a homepage dynamically listing out all nodes in your current cluster, with related information including internal IP addresses, CPU usage, memory usage, and file systems usage. 
+![kubernetes UI home page](k8s-ui-overview.png)
+
+### Dashboard Views
+Click on "Views" button in the top-right of the page to see other views available, which include: Explore, Pods, Nodes, Replication Controllers, Services, and Events. 
+
+#### Explore View 
+The "Explore" view allows your to see the pods, replication controllers, and services in current cluster easily.  
+![kubernetes UI Explore View](k8s-ui-explore.png)
+"Group by" dropdown list allows you to group these resources by a number of factors, such as type, name, host, etc.
+![kubernetes UI Explore View - Group by](k8s-ui-explore-groupby.png)
+You can also create filters by clicking on the down triangle of any listed resource instances and choose which filters you want to add.
+![kubernetes UI Explore View - Filter](k8s-ui-explore-filter.png)
+To see more details of each resource instance, simply click on it.  
+![kubernetes UI - Pod](k8s-ui-explore-poddetail.png)
+
+### Other Views
+Other Views (Pods, Nodes, Replication Controllers, Services, and Events) simply list out related information of each resource. You can also click on any instance for more details. 
```

Comment:
Unless "Other Views" is the label of a button or some other element in the UI, you should not capitalize the V in views.

Also I would maybe say "simply list information about each type of resource"